docs/release-notes.txt

```diff
@@ -4,6 +4,115 @@
  Release Notes
 ###############
 
+**************
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**New Features**
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
+   easily train object detection models. MMDetection provides efficient implementations of popular
+   object detection methods like Mask R-CNN, Faster R-CNN, and DETR on par with Detectron2. In
+   addition, cutting-edge approaches from academia are regularly added to the library.
+
+-  Add a new ``--auto-work-dir`` feature to ``det deploy local``. Setting ``--auto-work-dir
```

Comment:
question: Any documentation for the "Note that containers run as the root user by default,
   so you may want to configure your user with ``det user``..." case available?

Reply:
We have this: https://docs.determined.ai/latest/topic-guides/cluster-configuration/users.html#running-tasks-as-particular-agent-users

Reply:
Done!

Reply:
Can you let it start with `Deploy: `